test/options-manager.js

```diff
@@ -457,6 +457,15 @@ test('buildConfig: parserOptions', t => {
 	t.is(config.baseConfig.parserOptions.sourceType, 'script');
 });
 
+test('buildConfig: prevents useEslintrc option', t => {
+	t.throws(() => manager.buildConfig({
```

Comment:
I prefer not using inline arrow function when the return value is not used. It just makes it harder to read the code.